src/main/webapp/app/exercises/shared/plagiarism/plagiarism-split-view/text-submission-viewer/text-submission-viewer.component.ts

```diff
@@ -53,12 +53,12 @@ export class TextSubmissionViewerComponent implements OnChanges {
     /**
      * Token that marks the beginning of a highlighted match.
      */
-    public tokenStart = '<span class="plagiarism-match">';
+    private readonly tokenStart = '<span class="plagiarism-match">';
 
     /**
      * Token that marks the end of a highlighted match.
      */
-    public tokenEnd = '</span>';
+    private readonly tokenEnd = '</span>';
 
```

Comment:
Optional: Since this is only used for the string construction and not in the component, you could move these above the component as `const` variables.